scaladoc/src/dotty/tools/scaladoc/tasty/ClassLikeSupport.scala

```diff
@@ -525,7 +525,7 @@ trait ClassLikeSupport:
     experimental: Option[Annotation] = None
   ) = Member(
     name = symbol.normalizedName,
-    fullName = symbol.fullName,
+    fullName = if symbol.fullName endsWith "$" then symbol.fullName.dropRight(1) else symbol.fullName,
```

Comment:
I don't think that's correct. Classes in Scala can normally end in `$` i.e. you can declare `class A$`. You have to check that the class in question is a `module class`.

Reuse the logic from here:

https://github.com/lampepfl/dotty/blob/main/scaladoc/src/dotty/tools/scaladoc/tasty/NameNormalizer.scala#L13-L19

and add a new method `normalizedFullName` there.

Reply:
Okay, thank you, I correct that